ui/src/table.tsx

```diff
@@ -153,6 +153,10 @@ export interface Table {
   pagination?: TablePagination
   /** The events to capture on this table. One of 'search' | 'sort' | 'filter' | 'download' | 'page_change' | 'reset' | 'select'. */
   events?: S[]
+  /** True to allow only on row to be selected at time. Mutually exclusive with `multiple` attr. */
+  isSingle?: B
```

Comment:
The public API should be snake-cased. However, let's use `single` instead of `is_single` for this case to be consistent with `multiple`.